src/tests/evil/evil_test_stdlib.c

```diff
@@ -205,6 +205,34 @@ EFL_START_TEST(evil_stdlib_mkstemp)
 }
 EFL_END_TEST
 
+EFL_START_TEST(evil_stdlib_mkstemp_create_many)
+{
+   const unsigned long long files_to_create = 1000;
```

Comment:
Why 1000?

Reply:
First I thought to create as many files as possible, trying to reach the posix limit, but that is completely nonsense: it will take _forever_ to finish, possibly reaching timeout. 
Than I put some arbitrary quantity to be described as 'many'. :no_good:

Reply:
welp...probably there are few cases where people create as many as 1k temporary files, sooooo, fair enough

Reply:
Maybe it should be dynamically allocated?

Reply:
Also, any specific reason for `unsigned long long`? If it is to conform with array length type, just use an straight forward `size_t`.